internal/provider/types/datasource/services.go

```diff
@@ -98,6 +98,12 @@ var HealthCheckPath = schema.StringAttribute{
 	MarkdownDescription: "If you're running a server, enter the path where your server will always return a 200 OK response. We use it to monitor your app and for [zero downtime deploys](https://docs.render.com/deploys#zero-downtime-deploys).",
 }
 
+var MaxShutdownDelaySeconds = schema.Int64Attribute{
+	Computed:            true,
+	Description:         "The maximum amount of time (in seconds) that Render waits for your application process to exit gracefully after sending it a SIGTERM signal.",
+	MarkdownDescription: "The maximum amount of time (in seconds) that Render waits for your application process to exit gracefully after sending it a SIGTERM signal.",
```

Comment:
We only need the `MarkdownDescription` if it includes markdown that's not used in the `Description`